config/kubernetes/sig-release/teams.yaml

```diff
@@ -169,34 +170,38 @@ teams:
     - spiffxp # subproject owner
     members:
     - aishsundar # subproject owner
-    - alejandrox1 # 1.15 CI Signal Lead
+    - alejandrox1 # 1.16 CI Signal Lead
     - alenkacz # 1.15 CI Signal Shadow
-    - bubblemelon # 1.15 Branch Manager
+    - bubblemelon # Branch Manager
     - castrojo # 1.15 Communications Lead
-    - claurence # 1.15 RT Lead
+    - claurence # subproject owner
     - craiglpeters # 1.15 Enhancements Shadow
     - daminisatya # 1.15 Docs Shadow
     - dstrebel # 1.15 Bug Triage Shadow
-    - idealhack # 1.15 Branch Manager Shadow
+    - guineveresaenger # 1.16 RT Lead Shadow
+    - idealhack # Branch Manager
     - imkin # 1.15 Test Infra Lead
     - jberkus # subproject owner
-    - jeefy # 1.15 Release Notes Shadow
+    - jeefy # 1.16 RT Lead Shadow
     - jimangel # 1.15 CI Signal Shadow
     - justaugustus # subproject owner
-    - kacole2 # 1.15 Enhancements Lead
+    - kacole2 # 1.16 Enhancements Lead
     - Katharine # 1.15 Test Infra Shadow
     - lachie83 # 1.15 RT Lead Shadow
     - MAKOSCAFEE # 1.15 Docs Lead
     - mariantalla # 1.15 Test Infra Shadow
     - nickchase # 1.15 Release Notes Shadow
-    - nikopen # 1.15 RT Lead Shadow
+    - nikopen # 1.16 RT Lead Shadow
+    - onlydole # 1.16 Communications Lead
     - onyiny-ang # 1.15 Release Notes Lead
+    - saschagrunert # 1.16 Release Notes Lead
+    - simplytunde # 1.16 Docs Lead
     - slicknik # 1.15 Branch Manager Shadow
     - smourapina # 1.15 CI Signal Shadow
     - soggiest # 1.15 Bug Triage Lead
```

Comment:
You should remove me from Bug Triage lead

Reply:
@soggiest -- planning a follow-up PR once we lock the shadows in. You'll probably be shuffling teams, so it doesn't make sense to remove you just yet :)